### PR TITLE
Update proof instructions for version 22

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Metamath-lamp now does automatic simple unification by default. This eliminates many steps. Modify the 2+2=4 section to match how the UI works now.

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
This now fixes #146.

**Comment by david-a-wheeler** [comment]
I am merging now to fix the inconsistencies.

File: docs/index.md [hunk 1]
```diff
@@ -525,17 +526,28 @@ sometimes prove many statements automatically.
...
 > and press Enter (Return).
-> Now, while no steps are selected, click on the
-> icon <img width="16" height="16" src="hub.svg" alt="Unify"> (unify).
-> Since there
-> was no specific step selected, it will try to justify all steps.
 
-Metamath-lamp will succeed in finding a justification for our new step,
+By default Metamath-lamp automatically runs a simple unification process
+to justify any steps that aren't yet justified.
+Though this process,
```
Comment (expln) [inline docs/index.md:532]: typo `Though` -> `Through`

File: docs/index.md [hunk 2]
```diff
@@ -525,17 +526,28 @@ sometimes prove many statements automatically.
...
+Metamath-lamp will succeed in finding a justification for this new step,
 so it will show a green checkmark next to our new step.
 The justification it will show is `2 : oveq1i`, which means that it can
 justify this new statement by applying theorem `oveq1i` and providing step 2
 as the hypothesis required by `oveq1i`.
 
+Note: older versions of Metamath-lamp didn't automatically run a simple
+unification process. You can also disable this automatic execution of the
+unification process after each change in a step
+(go to and unselect *Automatically "Unify All"*).
```
Comment (expln) [inline docs/index.md:542]: typo `go to and` -> `go to Settings and`

**Review by expln — COMMENTED** [review]
I found only simple typos.

**Comment by expln** [comment]
> I am merging now to fix the inconsistencies.

Sure, please feel free to merge whenever you need. I am getting notifications about new PRs to my email, so I will review them sooner or later.

File: docs/index.md (hunk 1 above)
Comment (david-a-wheeler) [reply docs/index.md:532]: Thanks for reporting this! I'll fix it in a separate pull request.

**Pull request merged**